src/Chapter1.hs

```diff
@@ -636,7 +658,12 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = findFirst n
```

Comment:
As `firstDigit n = findFirst n` that means that these two functions are the same for us. So, you could add the body of `findFirst` in here, and not introduce additional functions in `where` 🙂